odoo/addons/base/models/res_config.py

```diff
@@ -544,7 +544,10 @@ def default_get(self, fields):
                         _logger.warning(WARNING_MESSAGE, value, field, icp)
                         value = 0.0
                 elif field.type == 'boolean':
-                    value = bool(value)
+                    if 'True' in value:
```

Comment:
There is a `odoo.tools.str2bool` function that you could use.

Reply:
Well well, thanks for enlighten me